rules/0131_test.go

```diff
@@ -170,7 +170,11 @@ func TestGetRequestMessageUnknownFields(t *testing.T) {
 	for _, test := range tests {
 		errPrefix := "AIP-131 Request Unknown Fields"
 		req, err := lint.NewProtoRequest(testutil.MustCreateFileDescriptorProto(
-			testutil.FileDescriptorSpec{Template: tmpl, Data: test},
+			testutil.FileDescriptorSpec{
+				Template:             tmpl,
+				Data:                 test,
+				AdditionalProtoPaths: []string{testdatadir("api-common-protos")},
```

Comment:
Why adding this?

Reply:
Good catch, I think i was originally using that test to test the common protos. removed.